tests/test_flask_pyoidc.py

```diff
@@ -890,3 +886,59 @@ def test_access_control_should_run_view_function_if_valid_token(self):
                 scopes_required=['read', 'write'])(view_mock)()
             assert view_mock.called
             assert flask._app_ctx_stack.top.current_token_identity == token_introspection_response
+
+    def test_get_url_for_logout_view(self):
```

Comment:
How are this and the next test `test_get_url_for_logout_view_should_raise_build_error_if_mounted_under_custom_endpoint`?
Aren't they more related to #124?

Reply:
This one is to increase test coverage. #124 came later.

Reply:
Increasing test coverage would be another issue better suited for a separate PR.